samples/subsys/modbus/rtu_server/overlay-cdc-acm.conf

```diff
@@ -0,0 +1,4 @@
+CONFIG_USB_DEVICE_STACK=y
+CONFIG_USB_DEVICE_PRODUCT="Zephyr CDC ACM MODBUS sample"
+CONFIG_SHELL_BACKEND_SERIAL_CHECK_DTR=y
```

Comment:
Why is that needed? The shell is not activated in this sample, causing a warning:

```
warning: SHELL_BACKEND_SERIAL_CHECK_DTR (defined at subsys/shell/backends/Kconfig.backends:75) was
assigned the value 'y' but got the value 'n'. Check these unsatisfied dependencies:
SHELL_BACKEND_SERIAL (=n), SHELL_BACKENDS (=n), SHELL (=n). See
http://docs.zephyrproject.org/latest/kconfig.html#CONFIG_SHELL_BACKEND_SERIAL_CHECK_DTR and/or look
up SHELL_BACKEND_SERIAL_CHECK_DTR in the menuconfig/guiconfig interface. The Application Development
Primer, Setting Configuration Values, and Kconfig - Tips and Best Practices sections of the manual
might be helpful too.
```